packages/empty-state/src/EmptyState.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+import * as React from "react";
+import Image, { thumbnailSizes } from "@crave/farmblocks-image";
+import Text, { fontTypes, fontSizes } from "@crave/farmblocks-text";
+import Button, { buttonTypes, buttonSizes } from "@crave/farmblocks-button";
+
+import Container from "./styledComponents/Container";
+
+type Props = {
+  title: string,
+  width?: number,
+  imageSrc?: string,
+  description?: string,
+  primaryButtonLabel?: string,
+  onPrimaryButtonClick?: (SyntheticEvent<HTMLButtonElement>) => void,
+  info?: string
+};
+
+class EmptyState extends React.Component<Props> {
+  static defaultProps = {
+    width: 560
```

Comment:
I think we don't need to set width here.
What about assume the full width and let the parent handle `width`?